project/src/components/film-buttons/film-buttons.tsx

```diff
@@ -1,8 +1,13 @@
 import { useNavigate } from 'react-router-dom';
 import { AppRoute } from '../../const/const';
 import { Link } from 'react-router-dom';
+import { PromoCardType } from '../../mocks/mocks';
 
-function FilmButtons():JSX.Element {
+type FilmButtonsProps ={
+  currentFilm: PromoCardType,
+}
+
+function FilmButtons({currentFilm}:FilmButtonsProps):JSX.Element {
```

Comment:
Вот так, или prettier заставлю поставить)
```suggestion
function FilmButtons({currentFilm}: FilmButtonsProps): JSX.Element {
```